File: Lab2/pzpi-22-7-bystrytska-anastasiia-lab2/models/Room.js
const mongoose = require('mongoose');
const roomSchema = new mongoose.Schema({
  name: { type: String, required: true },
  description: { type: String, default: '' },
  temperature: { type: Number, default: null },
  humidity: { type: Number, default: null },
  updatedAt: { type: Date, default: Date.now },
});
module.exports = mongoose.model('Room', roomSchema);

File: Lab5/pzpi-22-7-bystrytska-anastasiia-lab5/mobile/data/model/Room.kt
package com.example.mobile.data.model

import java.util.Date

data class Room(
    val _id: String,
    val name: String,
    val description: String? = null,
    val temperature: Double? = null,
    val humidity: Double? = null,
    val updatedAt: String? = null // Змінюємо на String для простоти
)